### ISSUE TITLE
Blocked download

### ISSUE THREAD

**Issue by AHIGHROLLER** [exploitagency/vanitygen-plus#210]
Currently, Google Chrome Version 78.0.3904.108 (Official Build) (64-bit) will not allow/blocks the "windows-binary.zip" file. [https://support.google.com/chrome/answer/6261569?visit_id=637108993283798630-385620541&p=ib_download_blocked&hl=en&rd=1](url)

**Comment by Radexito** [comment]
Yep, using Brave 1.23.75 and win 10. defender blocked and detected "Win32/Tiggre!rfn" which is a trojan downloader. hmmm fishy.
